kong/db/schema/typedefs.lua

```diff
@@ -529,7 +540,7 @@ typedefs.no_hosts = Schema.define(typedefs.hosts { eq = null })
 
 typedefs.paths = Schema.define {
   type = "array",
-  elements = typedefs.path {
+  elements = typedefs.router_path {
```

Comment:
should we also duplicate `typedef.paths` to `typedef.router_paths` ? to be safe?

also the check below; the `match_none` is a duplicate, since it is already in the typedef of the individual elements

Reply:
I searched though the code and find no other use of paths. Maybe we could rename(instead of duplicate).

Reply:
I searched though the code and find no other use of paths. Maybe we could rename(instead of duplicate).